tests/Test.js

```diff
@@ -418,4 +421,29 @@ const JSON_WINDOWSTATUS = Test.JSON_WINDOWSTATUS = {
     [WindowStatus.KEY_STATE]: Test.JSON_WINDOW_STATE,
 };
 
+const GENERAL_SEAT_LOCATION = Test.GENERAL_SEAT_LOCATION = new SeatLocation()
+    .setGrid(Test.GENERAL_GRID);
+
+const JSON_SEATLOCATION = Test.JSON_SEATLOCATION = {
+    [SeatLocation.KEY_GRID]: Test.JSON_GRID,
+};
+
+const GENERAL_SEAT_STATUS = Test.GENERAL_SEAT_STATUS = new SeatStatus()
+    .setSeatLocation(Test.GENERAL_SEAT_LOCATION)
+    .setConditionActive(Test.GENERAL_BOOLEAN);
+
+const JSON_SEATSTATUS = Test.JSON_SEATSTATUS = {
+    [SeatStatus.KEY_SEAT_LOCATION]: Test.JSON_SEATLOCATION,
+    [SeatStatus.KEY_CONDITION_ACTIVE]: Test.GENERAL_BOOLEAN,
+};
+
+const GENERAL_SEAT_OCCUPANCY = Test.GENERAL_SEAT_OCCUPANCY = new SeatOccupancy()
+    .setSeatsOccupied([Test.GENERAL_SEAT_STATUS])
```

Comment:
Given that `seatsOccupied` and `seatsBelted` are arrays in the 'SeatsOccupancy.js' struct, does the `GENERAL_SEAT_STATUS` on line 431 need to be instantiated differently?

For example on line 128 `imageTypeSupported` is an array and is set up with the `GENERAL_FILETYPE_LIST` using the line `GENERAL_IMAGEFIELD.setImageTypeSupported(GENERAL_FILETYPE_LIST);`.

Reply:
Hi @santhanamk
I've created GENERAL_SEAT_STATUS_LIST and JSON_SEATSTATUS_LIST arrays to make code look cleaner.